en/messages.json

```diff
@@ -244,12 +248,15 @@
     "popupCriticalEmailForwardingHeadline": {
         "message": "Critical email forwarding"
     },
-    "popupBlockPromotionalEmailsHeadline_2": {
+    "popupPromoBlockingButtonTitle": {
```

Comment:
@peiying2 Is this okay to rename IDs?

Reply:
Renaming the ID will generate a new string - I think it behaves the same like the Fluent file. However, if it is localized already, it will show up as a prompt as 100% match in the TM. Localizers simply can copy and paste it over.